experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -63,13 +61,20 @@ include(
         "experiments/integrated/fluxnet/fluxnet_simulation.jl",
     ),
 )
-
-include(
-    joinpath(
-        climaland_dir,
-        "experiments/integrated/fluxnet/met_drivers_FLUXNET.jl",
-    ),
-)
+start_date = DateTime(2010) + Hour(time_offset)
+(; atmos, radiation, LAI, maxLAI, DATA_DT) =
```

Comment:
I need to get maxLAI and DATA_DT in another way.

Ideally just open file once and get the columns we need (transformed)?